.kokoro/nightly/integration.cfg

```diff
@@ -6,11 +6,6 @@ env_vars: {
   value: "gcr.io/cloud-devrel-kokoro-resources/java8"
 }
 
-env_vars: {
```

Comment:
@kolea2 Looks like we need to ignore this file for template regeneration for now.

Reply:
will do!